components/StoryCard/StoryCard.tsx

```diff
@@ -48,12 +48,11 @@ export const StoryCard = ({
   className,
   ...props
 }: StoryCardProps) => {
-  let headingSize: FontSizeType = 3;
-  if (isHorizontal) {
-    headingSize = 'f5';
-  } else if (isSmallHeading && !isHorizontal) {
-    headingSize = 2;
-  };
+  // The heading level of the tags should be one more than the heading level of the card, but maxes out at h6
+  const tagsHeadingLevelNumber: HeadingLevelNumberType =
+    Math.min(parseInt(headingLevel?.slice(1), 10) + 1, 6) as HeadingLevelNumberType;
+
+  const tagsHeadingLevel: HeadingType = `h${tagsHeadingLevelNumber}`;
```

Comment:
Add a screen reader heading for the story tags, which is one level down compared to the heading level of the Card heading, but it needs to max out at h6 (which is unlikely scenario anyway that we'll actually use a h6 for the card heading)

I'm trying my best with TypeScript but there is no way for me to get around not using the as HeadingLevelNumberType here. I think this is robust enough for this project though.

Reply:
I was looking to see if we can set an Integer Range type and the best answer I found was here: https://stackoverflow.com/questions/39494689/is-it-possible-to-restrict-number-to-a-certain-range

Reply:
I guess the refactor here would be to change the `headingLevel` prop from a set of strings to the integer range of 1 - 6 and then to construct the heading level string to pass down into the `<Heading>` component.

Reply:
The range restriction is indeed cool 😄 , but, I might cheat a bit for this heading level here since for what we use it for (or just holding off for a bit till we're done with the story nav feature), it would be very unlikely that we run into any issues.

Reply:
Yeah, that is totally fine here.